source/codegen/templates/service_helpers.mako

```diff
@@ -101,41 +101,48 @@ ${initialize_standard_input_param(function_name, parameter)}\
 </%def>
 
 ## Initialize an enum input parameter for an API call.
-<%def name="initialize_enum_input_param(function_name, parameter)">\
+<%def name="initialize_enum_input_param(function_name, parameter, enums)">\
 <%
+  config = data['config']
```

Comment:
You pull `config` out of `data` here. Why not do the same with `enums` instead of passing it in as a param?